content/general/concepts/binary-search-tree/binary-search-tree.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Binary Search Tree'
+Description: 'A binary search tree is a data structure that is comprised of nodes in a branching relationship, each node having a key signifying its value.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Conceptual'
+  - 'Binary Tree'
+  - 'Data Structures'
+CatalogContent:
+  - 'complex-data-structures'
+  - 'paths/computer-science'
+---
+
+A **binary search tree** is a data structure that is comprised of nodes in a branching relationship, each node having a key signifying its value. Each node can have zero, one, or two child nodes branching off from it. The tree has one root node that has no parent.
+
+All nodes in the left branch of a parent node have values that are less than the parent node's value, and all nodes in the right branch of a parent node have values that are greater than the parent node's value.
+
+Below is a diagram of a binary search tree.
+
+![Binary Search Tree](https://raw.githubusercontent.com/Codecademy/docs/main/media/binary-tree-labeled.png 'Binary Search Tree Diagram')
+
+The part of the tree with nodes `1`, `2` & `3` is referred to as the left subtree, and the part with nodes `5`, `6` & `7` is referred to as the right subtree. Both subtrees are themselves binary search trees. The nodes `1`, `3`, `5` & `7` are called leaf nodes because they have no children.
```

Comment:
```suggestion
The part of the tree with nodes `1`, `2`, and `3` is referred to as the left subtree, and the part with nodes `5`, `6`, and `7` is referred to as the right subtree. Both subtrees are themselves binary search trees. The nodes `1`, `3`, `5`, and `7` are called leaf nodes because they have no children.
```

In this context, since we're in a paragraph as opposed to a title/sub-header, I believe we should stick with an oxford comma followed a spelled-out "and".